lib/yaml/constructor.py

```diff
@@ -76,9 +81,23 @@ def construct_object(self, node, deep=False):
         self.recursive_objects[node] = None
         constructor = None
         tag_suffix = None
-        if node.tag in self.yaml_constructors:
+
+        if (hasattr(self, '_yaml_constructors') and
+            node.tag in self._yaml_constructors and
+            self._yaml_constructors[node.tag] is not None
```

Comment:
Could be easier to delete the constructor when doing `add_constructor(tag, None)`? Then this check wouldn't be necessary